profiles/arch/amd64-fbsd/clang/make.defaults

```diff
@@ -6,3 +6,6 @@ CFLAGS="-O2 -pipe"
 CXXFLAGS="-stdlib=libc++ ${CFLAGS}"
 FFLAGS="${CFLAGS}"
 FCFLAGS="${CFLAGS}"
+
+# sys-devel/clang require static-analyzer, bug 539858
+BOOTSTRAP_USE="${BOOTSTRAP_USE} static-analyzer"
```

Comment:
Why `BOOTSTRAP_USE`? Wouldn't it be better to just `package.use.force` it on LLVM, like you do for clang?